app/src/main/kotlin/com/hedvig/app/service/push/PushNotificationService.kt

```diff
@@ -34,10 +35,12 @@ class PushNotificationService : FirebaseMessagingService() {
       )
       .build()
     WorkManager.getInstance(this).enqueue(work)
```

Comment:
I wonder how `PushNotificationWorker` fits in this scenario now. And if that one can be removed completely, as I see it does two mutations: `RegisterPushTokenMutation`, `NotificationRegisterDeviceMutation`. Maybe those two aren't even needed anymore? It even has this bug there right now, where if there's no token it simply retries forever, [here](https://github.com/HedvigInsurance/android/blob/bf87684a61daab10d17174076f276bf4519d7e1a/app/src/main/kotlin/com/hedvig/app/service/push/PushNotificationWorker.kt#L31-L33) if we can get rid of it altogether it'd be great. If not it'd be nice to help me understand what it does and it needs to stay.

Reply:
Good idea. I will investigate if its possible to remove this!